contracts/ParsecBridge.sol

```diff
@@ -35,7 +36,9 @@ contract ParsecBridge is PriorityQueue {
   uint256 public averageGasPrice; // collected gas price for last submitted blocks
   uint256 exitDuration;
   bytes32 public tipHash; // hash of first period that has extended chain to some height
-  ERC20 public token;
+
+  mapping(uint256 => PriorityQueue.Token) public tokens;
```

Comment:
Should color be restricted to uint16 there?

Reply:
maybe also for slots?